### DISCUSSION TITLE
Feature Request: better search

### DISCUSSION THREAD

**Discussion by MaxWasUnavailable** [subrotokumar/kurumi#7] [Ideas]
It would be nice to have better filters & sorting for the search system.

For example: filtering on tags/genres, airing/not airing, etc...

Likewise, it would be nice to have sorting options. Sort by release date, score, popularity (watch count?), etc...

**Comment by subrotokumar** [comment]
Hello @MaxWasUnavailable,

Your suggestion for improved filters and sorting for our search system is greatly appreciated. The idea of enhancing user experience through better tag and genre filters, along with sorting options like release date, score, and popularity, aligns well with our commitment to making our app more user-friendly.

I value your feedback and understand the importance of providing comprehensive tools to navigate and discover content more efficiently. While I can't promise immediate implementation, I'm taking your input into serious consideration. We will explore ways to incorporate these features to enhance your overall experience.

Thank you for taking the time to share your thoughts. Your contribution is vital in shaping the future of our app. Please keep an eye out for updates and feel free to participate in ongoing discussions.

**Comment by subrotokumar** [comment]
In the upcoming version 1.4.0, we'll be introducing additional filter options to the Media Search Screen. You'll now have the ability to filter content by tags and genres, giving you more control over your search results.
Update Release Date: 15-08-2023 6:00 AM IST
